controllers/routemonitor/routemonitor_controller_supplement.go

```diff
@@ -290,8 +476,9 @@ func (r *RouteMonitorReconciler) templateForServiceMonitorResource(routeMonitor
 	return serviceMonitor
 }
 
-// commonTemplateLables creates a set of common labels to most resources
-// this function is here in case we need more labels in the future
-func commonTemplateLables() map[string]string {
-	return map[string]string{"app": blackBoxName}
+// templateForServiceMonitorName return the generated name from the RouteMonitor.
```

Comment:
is it a template then? Wouldn't be something like `generateServiceMonitorName` be more appropriate?

Reply:
I am not 100% with naming so will take your name for now